src/components/pageHeader/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { Icon } from '@components/icon';
+import { cn } from '@utils/helpers';
+
+interface IPageHeader {
+  title: string;
+  blockNumber?: number;
+}
+
+export const PageHeader = (props: IPageHeader) => {
```

Comment:
This looks more like subpage header(it is not applicable for all the pages).

Reply:
one more think ... if we need to have diff behavior for goBack ... we can use router state for custom backURL with default `-1` because in some cases we want to go back to explorer in other to go back to view all page for finalized block etc.